docs/source-app/glossary/storage/drive_content.rst

```diff
@@ -10,7 +10,13 @@ The Drive object provides a central place for your components to share data.
 
 The Drive acts as an isolate folder and any component can access it by knowing its name.
 
-Your components can put, list, get, and delete files from and to the Drive (except LightningFlows).
+Drives offer two types of storage: temporary and persistent.
```

Comment:
Let's align on these definitions. What we call Temporary here are actually persistent across app restarts, so this nomenclature might generate confusion.

The main difference here is that standard drives are accessible through the Drive API from a Work, while you can mount S3 locations RO to a Work at a filesystem mount point.

This also means that we want to enable mounting RW ("Temporary") drives as RO mount points ("Persistent"), which will make these definitions not work.

Reply:
@lantiga please take peek. I've made changes based on your feedback.